refactor(preparations): Rewrite `/preparations/` pages (#819)
discordjs/guide: guide/preparations/setting-up-a-bot-application.md

client.login(process.env.TOKEN);
```

Another common approach is storing these values in a file called `.env` (some hosting solutions automatically ignore `.env` files and load them into your process. We will shortly cover how to keep them safe from git's greedy tracking in a later section below). This approach is less prone to typos and spares you from always copying tokens into your command line. Each line in this file will hold a key-value pair separated by the `=` character.

```
TOKEN=NzkyNzE1NDU0MTk2MDg4ODQy.X-hvzA.Ovy4MCQywSkoMRRclStW4xAYK7I
A=123
B=456
```

> To load variables from a file into the process, you can use the [`dotenv` package](https://www.npmjs.com/package/dotenv). Install it in your project directory via `npm install dotenv`. You can then require and use the package to load your `.env` file and attach the variables to `process.env`:

Reviewer: ```suggestion
You can use the [`dotenv` package](https://www.npmjs.com/package/dotenv) to load variables from a file into the process. Install it in your project directory by running the following command:

```sh:no-line-numbers
npm install dotenv
\```

You can then require and use the package to load your `.env` file and attach the variables to `process.env`:
```

- Note: The `\` under the code-block should be removed.
